samples/v1.3/Scenarios/CafeteriaMenu.json

```diff
@@ -0,0 +1,831 @@
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "Container",
+            "bleed": true,
+            "backgroundImage": {
+                "url": "https://raw.githubusercontent.com/pnp/AdaptiveCards-Templates/main/samples/visual-list/assets/cafe.jpg"
+            },
+            "items": [
+                {
+                    "type": "Container",
+                    "minHeight": "120px"
```

Comment:
Parsing this card failed during PR validation because shared model parser detected missing required properties.
I noticed that Container is missing 'items'. You may want to add empty items[], and try again.

Reply:
I have no idea what that means or where the empty items[] should be added. Here's how this would look in the site https://adaptivecards.io/designer/index.html?card=https%3A%2F%2Fraw.githubusercontent.com%2Fpnp%2FAdaptiveCards-Templates%2Fmain%2Fsamples%2Fvisual-list%2Fac-qv-cafe.json

That's linked from the following GitHub location where we have the designs from Design team - https://github.com/pnp/AdaptiveCards-Templates

I'm not a specialist on the json editing and what should be fixed here, just providing the samples based on request from Matt. If these are not something which we don't want to expose in the official site - all good.